ping_identity/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "cdcdaa67-b07a-49ef-a700-863c97bd9dfa",
+  "app_id": "ping-identity",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Low-code orchestration for Datadog API in PingOne DaVinci",
+    "title": "Ping Identity - Datadog API Connector",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Ping Identity DaVinci Logo",
+        "image_url": "images/Logo-DaVinci-Color.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Connector Configuration",
+        "image_url": "images/datadogConfig.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Connector Capabilites",
+        "image_url": "images/datadogCapabilities.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Connector Capabilites Configuration",
+        "image_url": "images/datadogCapabilityConfig.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Datadog Connector in a flow snippet",
+        "image_url": "images/datadogFlow.png"
+      }  
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "author": {
+    "support_email": "tap-global@pingidentity.com",
+    "name": "Ping Identity",
+    "homepage": "pingidentity.com",
```

Comment:
homepage should read: https://www.pingidentity.com

Didn't know this would error.